src/txn/txn_rollback_to_stable.c

```diff
@@ -432,14 +432,20 @@ __rollback_abort_row_reconciled_page(
     if ((mod = page->modify) == NULL)
         return (0);
 
-    if (mod->rec_result == WT_PM_REC_REPLACE)
+    if (mod->rec_result == WT_PM_REC_REPLACE) {
         WT_RET(__rollback_abort_row_reconciled_page_internal(session, mod->u1.r.disk_image,
           mod->u1.r.replace.addr, mod->u1.r.replace.size, rollback_timestamp));
-    else if (mod->rec_result == WT_PM_REC_MULTIBLOCK) {
+
+        /* Mark the page as dirty to let the reconciliation happens again on the page. */
```

Comment:
I want to mark the page dirty again if the page has already reconciled. If we don't mark the page dirty, in case if we aborted some updates in the update chain, if we don't mark the page is dirty, we may use the already reconciled image during the eviction. To avoid this scenario, mark the page as dirty if there is a reconciled image exists already.